### PR TITLE
Main site

### PR DESCRIPTION
**Pull request by berenteb**
First version of the main site. Please note every suggestion and it will be implemented.
- The carousel is only a placeholder / base component. It is not ready, and should be replaced in another issue. It does the job for now.

### CODE REVIEW

File: src/main/client/src/components/@commons/ImageCarousel.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  const previousImage = () => {
+    const previousIndex = currentImageIndex - 1
+    setCurrentImageIndex(previousIndex < 0 ? images.length - 1 : previousIndex)
+  }
+  const nextImage = () => {
+    const nextIndex = currentImageIndex + 1
+    setCurrentImageIndex(nextIndex > images.length - 1 ? 0 : nextIndex)
+  }
+  if (images.length === 0) return null
+  return (
```
Comment (triszt4n) [inline src/main/client/src/components/@commons/ImageCarousel.tsx:20]: Maybe with `return (images.length === 0) && ...`

**Review by triszt4n — APPROVED** [review]
Sometimes it's alright using some outside package for things like carousels :D you don't have to create one by yourself. It's not beautiful right now, but it's fine

File: src/main/client/src/components/@commons/ImageCarousel.tsx (hunk 1 above)
Comment (berenteb) [reply src/main/client/src/components/@commons/ImageCarousel.tsx:20]: I don't think that is pussible. A boolean would be returned IMO, and it is not readable. Also you won't earn anything from that.

Comment (triszt4n) [reply src/main/client/src/components/@commons/ImageCarousel.tsx:20]: Yeah, boolean would be returned, but: https://stackoverflow.com/questions/64019668/what-type-should-be-used-on-a-react-component-that-returns-a-boolean-or-jsx

**Pull request merged**